src/main/java/org/opentestsystem/ap/imrt/common/model/BaseItem.java

```diff
@@ -0,0 +1,156 @@
+package org.opentestsystem.ap.imrt.common.model;
+
+import org.hibernate.annotations.Type;
+import org.hibernate.validator.constraints.NotBlank;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.DiscriminatorColumn;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import javax.persistence.OrderBy;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import java.time.Instant;
+import java.util.List;
+
+@Entity
+@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
+@Table(name = "item")
+@EntityListeners(AuditingEntityListener.class)
+@DiscriminatorColumn(name = "classification")
```

Comment:
I had to add a discriminator column to the table.  I couldn't find a way to have JPA do if stim use Stimulus class otherwise use item class.

So the classification table now has:
* stim 
* item

Reply:
Seems reasonable to me.